run.bash

```diff
@@ -36,8 +35,8 @@ fi
 test "$1" = "noclobber" && shift
 
 # try to find and activate the snakemake conda env if we need it
-if ! command -v 'snakemake' &>/dev/null && \
-   command -v 'conda' &>/dev/null && \
+# changed 2/13/2021 deleted snakemake detection conditional because snakmake 5.5.3 is installed on purves globally
```

Comment:
I'm not sure if this should be committed. On one hand, it's probably a good idea to make this change so that other lab members don't run into this problem. On the other hand, if anyone else (outside of our lab) ever decides to use this code, they might have problems if this is merged in.
What do you think? This might also be a good thing to ask @mdonaldsonmatasci?
Maybe there's a way to straddle both options?